src/app/dashboard-widgets/create-work-item-widget/create-work-item-widget.component.ts

```diff
@@ -44,8 +44,13 @@ export class CreateWorkItemWidgetComponent implements OnInit {
         active: true
       }])
       .switchMap(filters => this.workItemService
-        .getWorkItems(100000, filters))
+        .getWorkItems(100000, filters)
```

Comment:
100000 is a silly number. 100 is Max you will get from backend

Reply:
Yeah very true. I'll change it.